lib/Client.js

```diff
@@ -23,6 +23,10 @@ merge(Client.prototype, {
     return this._makeRequest(options);
   },
 
+  getAbsolute: function(path) {
+    return this._makeRequest(path);
+  },
+
```

Comment:
I wonder if we should run a check here that the URL matches the `scheme` and `base` instance variables of the client? Think it'd be a good extra validation to prevent future mistakes when using this method.

Reply:
Yeah, moved this to an auto url scheme detect. Cleaner internal API as well.